connectors/connectors/rest/rest-swagger-connector/src/main/java/io/syndesis/connector/swagger/DelegateEndpointImpl.java

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.syndesis.connector.swagger;
+
+import java.util.Map;
+
+import org.apache.camel.CamelContext;
+import org.apache.camel.Consumer;
+import org.apache.camel.DelegateEndpoint;
+import org.apache.camel.Endpoint;
+import org.apache.camel.EndpointConfiguration;
+import org.apache.camel.Exchange;
+import org.apache.camel.ExchangePattern;
+import org.apache.camel.PollingConsumer;
+import org.apache.camel.Processor;
+import org.apache.camel.Producer;
+import org.apache.camel.component.connector.DefaultConnectorEndpoint;
+
+@SuppressWarnings("deprecation")
+final class DelegateEndpointImpl implements DelegateEndpoint {
```

Comment:
Why is the endpoint delegate required ? On a first sight it does not add any extra functionality over the original endpoint.

Reply:
With #465 it will no longer be required

Reply:
So should we wait with merging so that you add the update or shoud we merge now and do the update later ?

Reply:
Let me see if I can incorporate this easily